code/game/objects/items/gift.dm

```diff
@@ -121,3 +121,24 @@
 
 	var/gift_type = pick(possible_gifts)
 	return gift_type
+
+/obj/item/gift/capsule
+	name = "gachapon capsule"
+	desc = "A capsule from a vending machine. It could contain theoretically anything!"
+
+/obj/item/gift/capsule/get_gift_type()
+	var/static/list/obj/item/possible_gifts = null
```

Comment:
```suggestion
	var/list/obj/item/possible_gifts = null
```

Making this list static is pointless since you change it every time this proc is called based on `gacha_odds`